File: ios/RNStopAppModule.m
//
//  RNStopAppModule.m
//  RNStopAppModule
//
//  Copyright © 2021 Jan Hellsten. All rights reserved.
//

#import "RNStopAppModule.h"

@implementation RNStopApp

RCT_EXPORT_MODULE()

RCT_EXPORT_METHOD(stopApp)
{
    exit(0);
}

@end

File: ios/RNStopAppModule.swift
//
//  RNStopAppModule.swift
//  RNStopAppModule
//
//  Copyright © 2021 Jan Hellsten. All rights reserved.
//

import Foundation

@objc(RNStopAppModule)
class RNStopAppModule: NSObject {
  @objc
  func constantsToExport() -> [AnyHashable : Any]! {
    return ["count": 1]
  }

  @objc
  static func requiresMainQueueSetup() -> Bool {
    return true
  }
}
